firmware/src/main.cpp

```diff
@@ -1,35 +1,69 @@
-#include "Arduino.h"
-
-#include "Debug.h"
+/* **************************** Local Includes ***************************** */
 #include "mPMTTestStand.h"
+#include "Timer.h"
+#include "Debug.h"
 
-const mPMTTestStandIO io = {
-    // Serial Devices
-    .serial_comm            = Serial,
-    .serial_comm_baud_rate  = 115200,
-    // Thermistor Pins
-    .pin_therm_amb          = A0,
-    .pin_therm_motor1       = A1,
-    .pin_therm_mpmt         = A2,
-    .pin_therm_motor2       = A3,
-    .pin_therm_optical      = A4,
-    // Gantry X-Axis Pins
-    .pin_motor_x_step       = 5,
-    .pin_motor_x_dir        = 6,
-    .pin_motor_x_enc_a      = 7,
-    .pin_motor_x_enc_b      = 8,
-    .pin_motor_x_ls_home    = 9,
-    .pin_motor_x_ls_far     = 10,
-    // Gantry Y-Axis Pins
-    .pin_motor_y_step       = 22,
-    .pin_motor_y_dir        = 23,
-    .pin_motor_y_enc_a      = 24,
-    .pin_motor_y_enc_b      = 25,
-    .pin_motor_y_ls_home    = 26,
-    .pin_motor_y_ls_far     = 27
+/* **************************** System Includes **************************** */
+#include <Arduino.h>
+
+const mPMTTestStandConfig conf = {
+    .io = {
+        // Serial Devices
+        .serial_comm            = Serial,
+        .serial_comm_baud_rate  = 115200,
+        // Thermistor Pins
+        .pin_therm_amb          = A0,
+        .pin_therm_motor1       = A1,
```

Comment:
Can we make `pin_therm_motor1` into `pin_therm_motor_axis_x` and  `pin_therm_motor2` into `pin_therm_motor_axis_y` . I think I was the one to call them motor1 and 2 but it's unclear what they refer to looking back.

Reply:
How about just `pin_therm_motor_x` and `pin_therm_motor_y` (no `axis` to keep it shorter)

Reply:
sounds good to me!